api/server/server.go

```diff
@@ -351,10 +351,24 @@ func getContainersChanges(eng *engine.Engine, version version.Version, w http.Re
 	if vars == nil {
 		return fmt.Errorf("Missing parameter")
 	}
-	var job = eng.Job("container_changes", vars["name"])
-	streamJSON(job, w, false)
 
-	return job.Run()
+	d := getDaemon(eng)
+	cont, err := d.Get(vars["name"])
+	if err != nil {
+		return err
+	}
+
+	changes, err := cont.Changes()
+	if err != nil {
+		return err
+	}
+
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
Let's try to use WriteJSON() instead so if we need to make a global change to how to write stuff its all in one spot.